backend/routes/user_routes.py

```diff
@@ -46,6 +48,14 @@ async def verify_login_link(
     return {"token": login_token}
```

Comment:
The login_link field should be `NULL` after this route executed.

Also, I think we should keep the login code in the database and not the full link, but to me that is optional.

Reply:
https://github.com/akvo/rag-doll/pull/16/commits/d6552b0d5170154dba2d4b93a9ce550aaa1e9042